README.md

```diff
@@ -151,15 +151,28 @@ Kindly read our [Contributing Guide](CONTRIBUTING.md) to learn and understand ab
 
 ## Local Dev Testing
 
-Prerequisites: `helm` and `kubectl` binaries installed and operational.
+Prerequisites, binaries installed and operational:
+
+- [Go](https://go.dev/doc/install)
+- [Helm](https://helm.sh/docs/intro/install/)
```

Comment:
Helm and Kubectl are not required. The v1.0 binary has all the code compiled into it.